c10/cuda/CUDACachingAllocator.cpp

```diff
@@ -583,6 +697,15 @@ class DeviceCachingAllocator {
       process_events();
     }
 
+    // try to allocate according to plan
+    if (memory_planner.active) {
```

Comment:
try to avoid exposing member vars as public if not necessary. For this one, you can expose an `is_active` member method.

Reply:
Done. Thank you.